qt_dotgraph/src/qt_dotgraph/edge_item.py

```diff
@@ -33,10 +33,15 @@
 
 from .graph_item import GraphItem
 
-
```

Comment:
Please do not change whitespaces-only - especially since in this case PEP8 requires the two empty lines.